react-accordion-component/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from 'react';
+
+const languages = [
```

Comment:
I suggest you declare a type for the language items, and then declare this to be an array of that type. This will make type checking in the components that use this better, and it will make sure you don't mistype things when you define this array.